hawtio-system/src/main/java/io/hawt/web/auth/AuthenticationConfiguration.java

```diff
@@ -230,6 +249,77 @@ public boolean isKeycloakEnabled() {
         return keycloakEnabled;
     }
 
+    /**
+     * Initialize OIDC configuration, so it is available both in {@link AuthConfigurationServlet} and
+     * {@link io.hawt.web.filters.ContentSecurityPolicyFilter}.
+     */
+    public void configureOidc() {
```

Comment:
It is only one configuration method and it may be disabled (to skip httpclient configuration and potential fetching of OIDC metadata). So please keep it.